pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -347,6 +349,19 @@ def handle_slash_command(
     elif ident_prompt == '/exit':
         console.print('[dim]Exiting…[/dim]')
         return 0, multiline
+    elif ident_prompt == '/cp':
+        try:
+            parts = messages[-1].parts
+        except IndexError:
+            console.print('[dim]No output available to copy.[/dim]')
+        else:
+            text_to_copy = '\n'.join(part.content for part in parts if part.part_kind == 'text')
```

Comment:
I'd rather use 2 newlines here, and check whether part is an instance of `TextPart`